src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -121,23 +120,22 @@ public ShooterSubsystem(final ShooterHoodSubsystem shooterHood) {
     // Sets the left shooter motor to follow the right motor, and be inverted.
     follower_shooterMAXLeft.follow(leader_shooterMAXRight, true);
 
-    lastPos = shooterEncoder.getPosition();
-    lastTime = Timer.getFPGATimestamp();
+    this.tab = tab;
   }
 
   public void shoot() {
     // fast shooter wheel
     // run gate if both of "" are good
-
+    double dumbSetpoint = Utils.map(MoPrefs.getShooterPIDSetpoint(), -5500, 5500, -1, 1);
```

Comment:
magic numbers.

Instead of "dumb", perhaps "openLoop" is a better name choice.  There may be other options, but I would prefer an accurate description versus op-ed.